src/assertions/descendantsCount.js

```diff
@@ -0,0 +1,22 @@
+export default function descendantsCount ({ wrapper, markup, arg1, arg2=1, sig }) {
+  this.assert(
+      wrapper.hasDescendantsCount(arg1, arg2),
+      () => 'expected ' + sig + ' to have ' + arg2 + ' descendants #{exp} ' + markup(),
+      () => 'expected ' + sig + ' not to have ' + arg2 + ' descendants #{exp} ' + markup(),
+      arg1
+  )
+}
+
+function convenienceHelper(count) {
+  return function ({ wrapper, markup, arg1, sig }) {
+    const arg2 = count;
+    descendantsCount.call(this, { wrapper, markup, arg1, arg2, sig});
+  };
+}
+
+export const descendantsCountWrappers = {
```

Comment:
I followed sinon's example, it does end up being more BDD-like because you can do:

```
expect(wrapper).to.have.threeDescendants(identifier)
```

vs

```
expect(wrapper).to.have.hasDescendantsCount(identifier, 3)
```